README.md

```diff
@@ -112,6 +113,189 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
     
 ---  
 
+## Angular 
+1. IDEs
+
```

Comment:
Don't use numbers list, and use proper markdown for headings

For headings - `##`
For list - `-`